rockcraft.yaml

```diff
@@ -8,38 +8,36 @@ base: ubuntu:22.04
 platforms:
   amd64:
 
+run_user: _daemon_
+
+environment:
+  SPARK_HOME: /opt/spark
+  SPARK_CONFS: /etc/spark8t/conf
+  JAVA_HOME: /usr/lib/jvm/java-11-openjdk-amd64
+  PYTHONPATH: /opt/spark/python:/opt/spark8t/python/dist:/usr/lib/python3/dist-packages
+  PATH: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/spark:/opt/spark/bin:/opt/spark/python/bin:/opt/spark-client/python/bin
+  HOME: /var/lib/spark
+  KUBECONFIG: /home/spark/.kube/config
```

Comment:
this needs to be relocated to spark home

Reply:
Great catch! Thanks!